ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v2/page/body.html

```diff
@@ -0,0 +1,21 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can we create a test case of a form using this page component, and check if it renders fine ? This is currently missing in this PR

Reply:
added test cases